integrations/terraform-cloud/changelog/.gitignore

```diff
@@ -1 +1,2 @@
 !.gitignore
+*.md
```

Comment:
no need, we want to sometime, merge few fixes before we release a version, so multiple contributors could develop to the same release, that way we won't release on each fix. 

It is less common that this happens, but let's allow that